packages/ui/src/components/va-icon/VaIcon.stories.ts

```diff
@@ -1,12 +1,114 @@
-import { defineComponent } from 'vue'
-import VaIcon from './VaIcon.demo.vue'
+import VaIconVue from './demo/VaIconVue.vue'
+import { VaIcon } from './'
 
 export default {
   title: 'VaIcon',
   component: VaIcon,
+  tags: ['autodocs'],
 }
 
-export const Default = defineComponent({
+export const Default = () => ({
   components: { VaIcon },
-  template: '<VaIcon/>',
+  template: `
+    <div class="flex flex-col">
```

Comment:
Default should be just `<VaIcon name="book">`.

Let's move font-awesome / ionic to "icon fonts" story